graphql-dgs-subscriptions-sse/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/sse/DgsSSESubscriptionHandler.kt

```diff
@@ -59,6 +59,12 @@ class DgsSSESubscriptionHandler(private val dgsQueryExecutor: DgsQueryExecutor)
             return ResponseEntity.badRequest().body(emitter)
         }
 
+        if (! query.contains("subscription")) {
```

Comment:
Can't the query be parsed to check for the operation name instead of just doing a contains?

Reply:
the operation name is different. It applies to ```subscription ABC { ...}``` where ABC is the operation name.

Reply:
Do you mean using the SchemaParser and parsing the operation definition?

Reply:
Not `SchemaParser`, because it's about parsing the query, not the schema. But with a `contains`, a query like the following would be accepted: `query SomeQuery { membership { subscriptionId }}`, while it clearly shouldn't.